### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getGrammar' of undefined

### ISSUE THREAD

**Issue by adalenv** [eschutz/atom-console#2]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.14.3 x64
**Electron**: 1.3.13
**OS**: Microsoft Windows 7 Professional 
**Thrown From**: [atom-console](https://github.com/eschutz/atom-console) package 0.3.9

### Stack Trace

Uncaught TypeError: Cannot read property 'getGrammar' of undefined

```
At C:\Users\C.R.C\.atom\packages\atom-console\lib\cmd\lang-mode.js:26

TypeError: Cannot read property 'getGrammar' of undefined
    at /packages/atom-console/lib/cmd/lang-mode.js:14:28)
    at /packages/atom-console/lib/command.js:146:26
    at Array.forEach (native)
    at CommandHandler.executeConsoleCommand (/packages/atom-console/lib/command.js:144:49)
    at AtomConsoleView.sendCommand (/packages/atom-console/lib/atom-console-view.js:185:45)
    at /packages/atom-console/lib/atom-console.js:28:69)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
  5x -9:17.5.0 core:backspace (input.hidden-input)
     -9:15.9.0 core:confirm (input.hidden-input)
     -3:25 dev-live-reload:reload-all (input.hidden-input)
     -2:23.7.0 command-palette:toggle (div.panels-item)
     -2:21.1.0 core:paste (input.hidden-input)
     -2:19.9.0 editor:consolidate-selections (input.hidden-input)
     -2:19.9.0 core:cancel (input.hidden-input)
     -2:05.3.0 window:toggle-dev-tools (div.panels-item)
     -1:14.2.0 core:confirm (input.hidden-input)
 10x -1:02.0 core:move-left (input.hidden-input)
  2x -1:00.2.0 core:confirm (input.hidden-input)
     -0:31.5.0 atom-console:toggle (div.package-detail.panels-item)
     -0:29.2.0 core:paste (div.line)
     -0:27.5.0 atom-console:send-command (input.hidden-input)
```

### Non-Core Packages

```
atom-console 0.3.9 
jQuery 1.2.5 
```

**Comment by eschutz** [comment]
Hi adalenv,
Did this error happen when you tried to load atom, or when you tried to execute a command in atom-console?

**Comment by adalenv** [comment]
tried to execute

**Comment by eschutz** [comment]
This appears to be an issue in other atom packages too which hasn't found a solution. It's probably a bug in atom, so I'll work on a temporary workaround for now.

**Comment by eschutz** [comment]
I added a fix, try it now (note that any `lang-mode` command may not work in your case)

**Comment by eschutz** [comment]
Was this error raised when you did not have a file open in atom? You haven't said the steps to reproduce.

**Comment by eschutz** [comment]
Fixed as of version 0.3.11, if you are still receiving this error please raise another issue.
